config/initializers/action_cable_patch.rb

```diff
@@ -0,0 +1,45 @@
+ActiveSupport.on_load(:action_cable) do
+  require 'action_cable/subscription_adapter/postgresql'
+  ActionCable::SubscriptionAdapter::PostgreSQL.class_eval do
+    def new_connection
```

Comment:
This is the new method right?  I wonder if we can detect it already existing (when Matthew fixes it for real and we start using it) and warn so we remember to remove this monkey patch?